firmware/winterbloom_sol/sol.py

```diff
@@ -254,6 +254,20 @@ def __str__(self):
             self.gate_4,
         )
 
+    def set_cv(self, output, value):
+        output = output.lower()
+        if output not in ["a", "b", "c", "d"]:
+            raise ValueError("No such CV channel '{}'".format(output))
+        if output is "a":
```

Comment:
since you're planning on adding more of these, it's okay to be a little clever here:

```python
getattr(self, "_cv_" + output).value = value
```

Reply:
Thanks, I try to avoid such things if possible but it definitely does reduce the LOC.